src/sentry/api/endpoints/organization_config_integrations.py

```diff
@@ -18,4 +18,10 @@ def get(self, request, organization):
             providers, organization=organization, serializer=IntegrationProviderSerializer()
         )
 
+        if "provider_key" in request.GET:
+            serialized = list(filter(lambda d: d["key"] in request.GET["provider_key"], serialized))
+
+        if(len(serialized) == 0):
```

Comment:
@NisanthanNanthakumar you can always do `if ! serialized`. Empty lists are falsy, unlike in JS. Good to be my Pythonic :)